test/solveEquation/solveEquation.test.js

```diff
@@ -88,17 +86,32 @@ describe('solveEquation for =', function () {
     ['x^2 (x - 5)^2 = 0', 'x = [0, 0, 5, 5]'],
     ['x^2 = 0', 'x = [0, 0]'],
     ['(x+2)^2 -x^2 = 4(x+1)', '4 = 4'],
+    // Nth root support
+    // TODO: + or - for sqrt roots
+    ['x^2 - 2 = 0', 'x = nthRoot(2, 2)'],
+    ['x^2 = 1', 'x = 1'],
+    ['x^3 - 3 = 0', 'x = nthRoot(3, 3)'],
+    ['(x^2 - 2) (x^2 - 5) = 0', 'x = [nthRoot(2, 2), nthRoot(5, 2)]'],
+    ['(x^2 + 2) (x^3 - 7) = 0', 'x = [nthRoot(-2, 2), nthRoot(7, 3)]'],
+    ['x^2 + 1 = 0', 'x = nthRoot(-1, 2)'],
+    ['(y + 1)^2 = 1', 'y = 0'],
+    ['(y + 1)^3 = 8', 'y = 1'],
+    ['(2x + 1)^3 = 1', 'x = 0'],
+    ['(3x + 2)^2 = 2', 'x = nthRoot(2, 2) / 3 - 2/3'],
+    // TODO: fix this
+    // ['(3x + 2)^2 + 2 = 1']
     // TODO: fix these cases, fail because lack of factoring support, for complex #s,
     // for taking the sqrt of both sides, etc
```

Comment:
This comment is no longer right, because we do take the square root of both sides now, so any cases that were previously failing because of that shouldn't be anymore (and should be uncommented out)